eks/modules/services/wait_for_url.sh

```diff
@@ -1,14 +1,14 @@
 #! /bin/sh
-echo "Trying to verify ${url} is up"
+echo "Trying to verify $URL is up"
 code=0
 while [ $code -ne 200 ]
 do
     echo "connecting..."
     code=$(curl -k -o /dev/null -Isw '%{http_code}\n' --connect-timeout 5 --max-time 5 $URL)
-    if [[ "$code" == '200' ]]; then
+    if [ "$code" -eq 200 ]; then
         break;
     fi
-    echo "...but received status ${code}; will check again in a minute ..."
-    sleep 60
+    echo "...but received status $code; will check again shortly ..."
+    sleep 10
```

Comment:
UAA doesn't come up until about 20+ minutes into the CAP deploy and because of the asynchronous nature of the helm installs this script can kick in right after the helm installs have triggered so waiting for only 10s is lot of unnecessary polling but only gaining little less than a minute in triggering the final metrics install.